unitOneAssessmentPractice.js

```diff
@@ -3,39 +3,100 @@ let assert = require('assert')
 // Question One:
 
 // Write a function called double that doubles a number
-
-// Uncomment out the next line to test your solution
+function double(num) {
+  return num*2;
+}
+double(14);
+// // Uncomment out the next line to test your solution
 // runQ1Tests()
 
 // Question Two:
 
 // Write a function called containsSeven that returns whether or not a number has an sevens in it.
+function containsSeven(num) {
+ let newNum = num.toString();
+ for (let i = 0; i < newNum.length; i++) {
+   if (newNum[i] === "7") {
+    return true;
+   } else {
+    i++;
+   }
+ }
+}
+ containsSeven(5378);
 
 // Uncomment out the next line to test your solution
 // runQ2Tests()
 
 // Question Three:
 
 // Write a function called capitalizeTheAs the capitalizes all of the lowercase "a"s in a string leaving all other characters the same
+function capitalizeTheAs(str) {
+  for (let i = 0; i < str.length; i++) {
+    if (str[i] === "a") {
+    return str.replace("a", "A");
+    }
+  }
+}
+capitalizeTheAs("giraffe");
 
 // Uncomment out the next line to test your solution
 // runQ3Tests()
 
 // Question Four:
 // Write a function called largest that returns the largest value in an array
+function largest(arr) {
+  let newLargest = "";
```

Comment:
Since the expected out put is a number, it's better to set newLargest = 0 instead of an empty string.

Reply:
thanks! that makes sense.